src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -1152,10 +1154,10 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
           }
           @inline def warnNumericWiden(tpSym: Symbol, ptSym: Symbol): Unit =
             if (!isPastTyper) {
-              val isInharmonic = (
+              val isInharmonic =
                 tpSym == IntClass  && ptSym == FloatClass ||
                 tpSym == LongClass && (ptSym == FloatClass || ptSym == DoubleClass)
-              )
+
```

Comment:
Someone started the parens style to make sure expressions aren't accidentally split: add a blank line and it changes. I don't feel a need for parens around if-else. Here I'm more on the fence. I'll be more comfortable with doti-style indentation verification.